app/code/Magento/Webapi/Model/Authorization/TokenUserContext.php

```diff
@@ -179,7 +179,9 @@ protected function processRequest()
         if (!$token->getId() || $token->getRevoked() || $this->isTokenExpired($token)) {
             $this->isRequestProcessed = true;
 
-            return;
+            throw new \Magento\Framework\Exception\NoSuchEntityException(
```

Comment:
@XxXgeoXxX As we considered in our last talk, the changes are backward incompatible
Need to revert it (including tests) and move logic to GraphQL responsible area (check place where user id is set to graphql context)